__tests__/articles.test.js

```diff
@@ -236,3 +236,84 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200 should return comments objects for the corresponding article_id", () => {
+    const votePatch = {
+      inc_votes: 42,
+    };
+    const article_id = 1;
+
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(votePatch)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Object);
+        expect(articles).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: article_id,
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: 142,
+            article_img_url: expect.any(String),
+            body: expect.any(String),
+          })
```

Comment:
It would be nice to assert against the specific values here, we know what they are so we should put them in. It allows us to be confident that our patch hasn't had an effect on any other property.